projects/pcre2/build.sh

```diff
@@ -21,6 +21,8 @@
 make -j$(nproc) clean
 make -j$(nproc) all
 
+cp $SRC/pcre2_fuzzer.dict $SRC/pcre2_fuzzer.options $OUT/
```

Comment:
Could you also move this line to the end of the script to be executed after compilation of the fuzzer.